x-pack/plugins/uptime/server/lib/alerts/duration_anomaly.ts

```diff
@@ -63,62 +60,74 @@ const getAnomalies = async (
   );
 };
 
-export const durationAnomalyAlertFactory: UptimeAlertTypeFactory<ActionGroupIds> = (
-  _server,
-  _libs,
-  plugins
-) =>
-  uptimeAlertWrapper<ActionGroupIds>({
-    id: 'xpack.uptime.alerts.durationAnomaly',
-    name: durationAnomalyTranslations.alertFactoryName,
-    validate: {
-      params: schema.object({
-        monitorId: schema.string(),
-        severity: schema.number(),
-      }),
+export const durationAnomalyAlertFactory: UptimeAlertTypeFactory = (_server, libs, plugins) => ({
+  id: 'xpack.uptime.alerts.durationAnomaly',
+  name: durationAnomalyTranslations.alertFactoryName,
+  validate: {
+    params: schema.object({
+      monitorId: schema.string(),
+      severity: schema.number(),
+    }),
+  },
+  defaultActionGroupId: DURATION_ANOMALY.id,
+  actionGroups: [
+    {
+      id: DURATION_ANOMALY.id,
+      name: DURATION_ANOMALY.name,
     },
-    defaultActionGroupId: DURATION_ANOMALY.id,
-    actionGroups: [
-      {
-        id: DURATION_ANOMALY.id,
-        name: DURATION_ANOMALY.name,
-      },
-    ],
-    actionVariables: {
-      context: [],
-      state: [...durationAnomalyTranslations.actionVariables, ...commonStateTranslations],
-    },
-    minimumLicenseRequired: 'basic',
-    async executor({ options, uptimeEsClient, savedObjectsClient, dynamicSettings }) {
-      const {
-        services: { alertInstanceFactory },
-        state,
-        params,
-      } = options;
+  ],
+  actionVariables: {
+    context: [],
+    state: [...durationAnomalyTranslations.actionVariables, ...commonStateTranslations],
+  },
+  minimumLicenseRequired: 'platinum',
+  async executor({
+    params,
+    services: { alertWithLifecycle, savedObjectsClient, uptimeEsClient },
+    state,
+  }) {
+    const { anomalies } =
+      (await getAnomalies(plugins, savedObjectsClient, params, state.lastCheckedAt as string)) ??
+      {};
+
+    const foundAnomalies = anomalies?.length > 0;
 
-      const { anomalies } =
-        (await getAnomalies(plugins, savedObjectsClient, params, state.lastCheckedAt)) ?? {};
+    if (foundAnomalies) {
+      const monitorInfo = await libs.requests.getLatestMonitor({
```

Comment:
swapped the imported version of this request for the `libs.requests` version. Any reason we weren't using this to begin with? Don't want to break anything unintentionally.